### ISSUE TITLE
UG is lacking in visuals

### ISSUE THREAD

**Issue by 0liverkong** [0liverkong/pe#4]
It is expected that there is at least some diagrams to illustrate to the user how to use the application, but there is only two diagrams found. In particular, the example on the bottom of page 5 has functionality that is entirely different from that of the standard command line interface - it would be helpful to have screenshots for this portion.

![Screenshot 2024-04-19 at 4.58.12 PM.png](https://raw.githubusercontent.com/0liverkong/pe/main/files/f9618a34-5181-4334-857b-55ac50f03b55.png)

**Comment by nus-se-script** [comment]
# Team's Response
No details provided by team.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Not enough visuals for commands: edit, add-skill, delete-skill, etc
>
>**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
>![Screenshot 2024-04-19 at 5.23.21 PM.png](https://raw.githubusercontent.com/nus-cs2103-AY2324S2/pe/master/files/9c3cdc8c-f4b9-49cf-85d1-994a07998e80.png)
>
>
>There are no pictures here to show how the command looks like in the UI with the corresoponding output and therefore is too wordy
>
>
>
>
>
><hr>
><sub>[original: nus-cs2103-AY2324S2/pe-interim#5323] [original labels: severity.Medium type.DocumentationBug]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>Thank you for your suggestion. However, the length of the explanation in the user guide is already very minimal compared to the complexity of the app and the number of features.
>
>Considering the current user guide is highly understandable, adding more illustrations to the user guide would be of a low priority. Thus we will be marking this bug as "Not in scope" with low severity.
# Items for the Tester to Verify
## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
## :question: Issue response

Team chose [`response.NotInScope`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.VeryLow`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
